reference-architecture/gcp/ansible/playbooks/openshift-installer-common-vars.yaml

```diff
@@ -26,4 +26,9 @@ openshift_hosted_metrics_storage_kind: dynamic
 openshift_metrics_hawkular_nodeselector: { 'role': 'infra' }
 openshift_metrics_cassandra_nodeselector: { 'role': 'infra' }
 openshift_metrics_heapster_nodeselector: { 'role': 'infra' }
+openshift_logging_es_pvc_dynamic: true
+openshift_logging_es_cluster_size: '{{ infra_node_instance_group_size }}'
+openshift_logging_es_nodeselector: { 'role': 'infra' }
+openshift_logging_kibana_nodeselector: { 'role': 'infra' }
+openshift_logging_curator_nodeselector: { 'role': 'infra' }
```

Comment:
What about "openshift_logging_fluentd_nodeselector"?

Reply:
Fluentd should be on all nodes, shouldn't it? (That's the default).

Reply:
It is. By default it will be set to "logging-infra-fluentd=true" which is in all nodes, I just wanted to know if you wanted to customize it just in case.